src/components/patterns/link.tsx

```diff
@@ -60,6 +64,7 @@ interface DrupalLinkProps extends PropsWithChildren<any> {
 
 
 export const DrupalLink = ({url, title, style, children, ...props}: DrupalLinkProps) => {
+  console.log(url);
```

Comment:
remove console log

Reply:
Dang. I should have reviewed the changes here on github.